test/common/common/logger_test.cc

```diff
@@ -0,0 +1,12 @@
+#include "common/common/logger.h"
```

Comment:
We already have log_macros_test. Can we potentially merge into a single logger_test?

Reply:
Yup. We can merge